raml-parser-2/src/test/resources/org/raml/v2/parser/security-scheme/describedBy-with-exclusive-keys/input.raml

```diff
@@ -0,0 +1,11 @@
+#%RAML 1.0
```

Comment:
keep directory names all lowercase for consistency, prepositions may also be avoided in some cases:

`describedBy-with-exclusive-keys` -> `describedby-exclusive-keys`
`describedBy-with-queryString` -> `describedby-querystring`